ansible/roles/sufia/tasks/data-repo.yml

```diff
@@ -64,6 +64,6 @@
     ignore_errors: True
 
   become: yes
-  become_user: '{{ project_user }}'
+  become_user: '{{ project_runner }}'
```

Comment:
Can you talk about why this is `project_runner` and not `project_owner`? The images added above are (currently) owned by hydra, so they can't be deleted by railsapps.